lib/view/acknowledgements_view.dart

```diff
@@ -0,0 +1,118 @@
+    'Acknowledegements'
+  ];
+  static const _thanksHtml = '''
+  So many people have worked together to make this project happen, and helped out in so many ways 🥰
+<br/>
+<br/>
+Firstly a huge thanks to <a href='https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app#contributors'>all our contributors</a>; Our designers, developers and all the medical staff. Your work is what is making this project happen.
+<br/>
+<br/>
+The entire <a href='https://www.gdgmelbourne.com/'>GDG Melbourne</a> community for pulling together on this.
+<br/>
+<br/>
+Several companies who have provided support and employee time, including <a href='https://www.ittybittyapps.com/'>Itty Bitty Apps</a>, and <a href='https://www.artprocessors.net/'>Art Processors</a>.
+<br/>
+<br/>
+<a href='https://codemagic.io/'>Code Magic</a> and <a href ='https://manichord.com/'>Manichord</a> for additional build minutes and support.
+<br/>
+<br/>
+<a href='https://slack.com/'>Slack</a> for upgrading our workspace free of charge.
+<br/>
+<br/>
+<a href='https://apple.com/'>Apple</a> and <a href='https://google.com/'>Google</a> for help with expedited app reviews and listings.
+<br/>
+<br/>
+Those who have given helpful and timely advice, including Byron Teu.<br/>
+  ''';
+  @override
+  Widget build(BuildContext context) {
+    // Analytics set screen name, stays until another screen changes it
+    Analytics.analyticsScreen(Constants.analyticsAcknowledgementsScreen);
+
+    return Scaffold(
+        backgroundColor: AppColors.appBackground,
+        appBar: AppBar(
+          // Warning brightness interacts with SystemUiOverlayStyle
+          // See system_bars.dart comments
+          brightness: Brightness.light,
+          backgroundColor: AppColors.appBarBackground,
+          iconTheme: Styles.appBarIconTheme,
+          title: const Text(
+            Strings.acknowledegementsTitle,
+            style: Styles.textH5,
+          ),
+        ),
+        body: Padding(
+            padding: const EdgeInsets.fromLTRB(16, 0, 16, 16),
+            child: SingleChildScrollView(
+                child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    children: [
+                  ..._cards.map((e) {
+                    if (e is HomeCard) {
+                      return ReusableCard.fromData(
+                        card: e,
+                        height: 48,
+                      );
+                    }
+                    return Padding(
+                      padding: const EdgeInsets.fromLTRB(0, 24, 16, 0),
+                      child: Text(
+                        e.toString(),
+                        style: Styles.textH4,
+                      ),
+                    );
+                  }),
+                  Padding(
+                    padding: const EdgeInsets.only(top: 4),
+                    child: ReusableCardBase(
+                      child: [
+                        HtmlWidget(
```

Comment:
Ideally would prefer RichText to get styled links and paragraph breaks. But I think this is fine for now 👍

Reply:
You can style the links with `hyperlinkColor`. Looks like in the mock it's a slightly lighter blue.
Paragraph breaks match the design as far as I see, what are you referring to?

Reply:
Leave it for now because we use this approach everywhere.  This is bigger discussion of RichText vs html/markdown 🤣